runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/BpmnModeler/BpmnModeler.types.ts

```diff
@@ -37,6 +37,12 @@ export interface ModelerHTMLCanvasElement extends HTMLCanvasElement {
     zoom: (precition: string, scale: string) => void;
     viewbox: (opts?: ModelerViewboxOpts) => ModelerViewboxResult;
 }
+export interface AdditionalInfo {
+    executionInfo?: IProcess['executionInfo'];
+    isAttended?: IProcess['isAttended'];
+    isTriggerable?: IProcess['isTriggerable'];
+    system?: IProcess['system'];
+}
```

Comment:
I suggest to use `Pick<>` together with `Partial<>`

Reply:
whats the advantage of that approach?